README.md

```diff
@@ -39,6 +40,100 @@ Note that politeiawww does not provide HTML output.  It strictly handles the
+#### 4. Finding your configuration files path: 
+* To make things simpler during development is handy having a configuration file for your components. Specially `politeiad` and `politeiawww` so you don't have to pass the parameters every time you run it. 
+* You may find the configuration files under the path:  
+  - **Mac OS** `/Users/<username>/Library/Application Support/Politeiad/politeiad.conf`
+  - **Windows** `C:\Users\<username>\AppData\Local\Politeiad/politeiad.conf`
+  - **Ubuntu** `~/.politeiad/politeiad.conf`
+
+#### 5. Set the configuration files:
+  
+Add the following configurations:
+
+**politeiad.conf**: 
+
+    rpcuser=user
+    rpcpass=pass
+    testnet=true	
+
+
+**politeiawww.conf**:
+
+    rpchost=127.0.0.1
+    rpcuser=user
+    rpcpass=pass
+    rpccert="/Users/<username>/Library/Application 
+    Support/Politeiad/https.cert"
+    proxy=true
+    testnet=true
+
+**Attention:** The rpccert path is referenced on MacOs path. See step 4 for more OS paths.
+#### 6. Run Politeiad by simple running on your terminal:
+
+    politeiad
+
+#### 7. Download server identity to client:
+
+    politeia -v -testnet -rpchost 127.0.0.1 identity
+
+Accept default path by pressing `enter`.
+
+Result should look something like this:
+
+```
+FQDN       : localhost
+Nick       : politeiad
+Key        : dfd6caacf0bbe5725efc67e703e912c37931b4edbf17122947a1e0fcd9755f6d
+Identity   : 99e748e13d7ecf70ef6b5afa376d692cd7cb4dbb3d26fa83f417d29e44c6bb6c
+Fingerprint: medI4T1+z3Dva1r6N21pLNfLTbs9JvqD9BfSnkTGu2w=
+
+Save to /Users/<username>/Library/Application Support/Politeia/identity.json or ctrl-c to abort
+Identity saved to: /Users/<username>/Library/Application Support/Politeia/identity.json
+```
+
+#### 8. Run Politeiawww by simple running on your terminal:
+
+    politeiawww
+
+**Awesome! From this point you have your politeia server up running!**
+#### 9. Creating a Reference Client:
+* With politeiad and politeiawww running type on your terminal:
+```
+politeiawww_refclient
+```
+Result should look something like this:
+```
+=== Start ===
+Request: GET /
+Version: 1
+Route  : /v1
+CSRF   :
+
+Request: GET /v1/policy
+Request: POST /v1/user/new
+Request: GET /v1/user/verify/?email=2e645574ba5dcf42@example.com&verificationtoken=41b27466ea295a9fd9e521d2aa2e8fc7837d48441fb6af8106abc4ecd929c94d&signature=546509014b4257b47186944f3ba47ed6e7077e3862cf635b0f4b9350ab3cf6c24ff8d341f65ecabc51b8dbeb8098deaa2c26d3faf4edbd6a6aa2aec47cd92b0a
+Request: POST /v1/proposals/new
+[...]
+refclient run successful
+=== End ===
+```
+* The generated email is what gives you access to the PoliteiaGUI application where your password will be the email's username. Example: 
+```
+email: 2e645574ba5dcf42@example.com
+password: 2e645574ba5dcf42
+```
+#### 10. Elevating user permission with politeiawww_dbutil
+* This is allow you to access politeia as an admin
```

Comment:
I would change this to: This tool allows you to elevate a user in politeiawww to have admin permissions. You will have to shut down `politeiawww`, and then execute in your terminal: